prac_05/hex_colours.py

```diff
@@ -0,0 +1,13 @@
+COLOUR_NAME_TO_CODE = {"aqua": "#00ffff", "black": "#000000", "blue": "#0018a8", "brown": "#a52a2a",
+                       "burgundy": "#800020", "celadon": "#ace1af", "chocolate": "#d2691e",
+                       "citron": "#9fa91f", "copper": "#b87333", "cream": "#fffdd0"}
+
+print(COLOUR_NAME_TO_CODE)
+colour_name = input("Enter colour: ").lower()
+while colour_name != "":
+    if colour_name in COLOUR_NAME_TO_CODE:
```

Comment:
If-else seems to work fine, could this be enhanced with exception handling?

Reply:
Good Catch 👍